docker-compose-ledger-common.yml

```diff
@@ -0,0 +1,62 @@
+services:
+  ledger-envoy:
+    image: ghcr.io/scalar-labs/scalar-envoy:1.5.0
+    container_name: "scalardl-samples-ledger-envoy-1"
+    ports:
+      - "9901:9901"
+      - "50051:50051"
+      - "50052:50052"
+    depends_on:
+      scalar-ledger:
+        condition: service_started
+    environment:
+      - admin_access_log_path=/dev/stdout
+      - scalardl_address=scalar-ledger
+      - service_listeners=scalar-service:50051,scalar-privileged:50052
+      - envoy_tls=false
+    networks:
+      - scalar-network
+
+  cfssl-init:
```

Comment:
This is not directly related to this PR, but I have one question.

Is CFSSL necessary for running `scalardl-samples`? I think if users want to generate a private key and certificate by using this CFSSL server, I think we need to deploy it. However, now, we are using the private key and certificate under the `fixture/` directory. So, I want to confirm whether this CFSSL server is necessary or not.

If it's not mandatory, I think we can make it an optional component. For example, we can set the `profiles` configuration as follows:

```diff
$ git diff docker-compose-ledger-common.yml
diff --git a/docker-compose-ledger-common.yml b/docker-compose-ledger-common.yml
index 6a4481c..982dc41 100644
--- a/docker-compose-ledger-common.yml
+++ b/docker-compose-ledger-common.yml
@@ -23,6 +23,8 @@ services:
     # they start up at the same time (by commands like `docker-compose up`),
     # they both attempt to generate the same missing certificate files, which
     # results in a race condition.
+    profiles:
+      - cfssl
     image: ghcr.io/scalar-labs/scalar-cfssl:1.0.0
     container_name: "scalardl-samples-cfssl-init-1"
     volumes:
@@ -32,6 +34,8 @@ services:
       - scalar-network

   cfssl-serve:
+    profiles:
+      - cfssl
     image: ghcr.io/scalar-labs/scalar-cfssl:1.0.0
     container_name: "scalardl-samples-cfssl-serve-1"
     volumes:
@@ -45,6 +49,8 @@ services:
       - scalar-network

   cfssl-ocspserve:
+    profiles:
+      - cfssl
     image: ghcr.io/scalar-labs/scalar-cfssl:1.0.0
     container_name: "scalardl-samples-cfssl-ocspserve-1"
     volumes:
```

In this case, the CFSSL server is not deployed by default as follows:

- Default behavior
  ```
  $ docker compose -f docker-compose-ledger-postgres.yml up -d
  ```
  ```
  $ docker compose -f docker-compose-ledger-postgres.yml ps
  NAME                               IMAGE                                             COMMAND                  SERVICE         CREATED         STATUS                        PORTS
  postgres-1                         postgres:15                                       "docker-entrypoint.s…"   postgres        2 minutes ago   Up About a minute (healthy)   0.0.0.0:5432->5432/tcp, :::5432->5432/tcp
  scalardl-samples-ledger-envoy-1    ghcr.io/scalar-labs/scalar-envoy:1.5.0            "/entrypoint.sh"         ledger-envoy    8 seconds ago   Up 4 seconds                  9001/tcp, 0.0.0.0:9901->9901/tcp, :::9901->9901/tcp, 10000/tcp, 0.0.0.0:50051-50052->50051-50052/tcp, :::50051-50052->50051-50052/tcp
  scalardl-samples-scalar-ledger-1   ghcr.io/scalar-labs/scalardl-ledger-byol:3.10.0   "./docker-entrypoint…"   scalar-ledger   8 seconds ago   Up 6 seconds (healthy)        50051/tcp
  ```

Also, if you want to deploy the CFSSL server, you can specify it by using the `--profile` option as follows:

- Specify the profile explicitly
  ```
  $ docker compose -f docker-compose-ledger-postgres.yml --profile cfssl up -d
  ```
  ```
  $ docker compose -f docker-compose-ledger-postgres.yml ps
  NAME                                 IMAGE                                             COMMAND                  SERVICE           CREATED              STATUS                        PORTS
  postgres-1                           postgres:15                                       "docker-entrypoint.s…"   postgres          3 minutes ago        Up 3 minutes (healthy)        0.0.0.0:5432->5432/tcp, :::5432->5432/tcp
  scalardl-samples-cfssl-ocspserve-1   ghcr.io/scalar-labs/scalar-cfssl:1.0.0            "docker-entrypoint.s…"   cfssl-ocspserve   10 seconds ago       Up 8 seconds                  8888/tcp, 0.0.0.0:8889->8889/tcp, :::8889->8889/tcp
  scalardl-samples-cfssl-serve-1       ghcr.io/scalar-labs/scalar-cfssl:1.0.0            "docker-entrypoint.s…"   cfssl-serve       10 seconds ago       Up 8 seconds                  0.0.0.0:8888->8888/tcp, :::8888->8888/tcp, 8889/tcp
  scalardl-samples-ledger-envoy-1      ghcr.io/scalar-labs/scalar-envoy:1.5.0            "/entrypoint.sh"         ledger-envoy      About a minute ago   Up About a minute             9001/tcp, 0.0.0.0:9901->9901/tcp, :::9901->9901/tcp, 10000/tcp, 0.0.0.0:50051-50052->50051-50052/tcp, :::50051-50052->50051-50052/tcp
  scalardl-samples-scalar-ledger-1     ghcr.io/scalar-labs/scalardl-ledger-byol:3.10.0   "./docker-entrypoint…"   scalar-ledger     About a minute ago   Up About a minute (healthy)   50051/tcp
  ```

If it's not a mandatory component, I think we can make it an optional thing as I mentioned above. This is because:

- We can reduce deployed components.
  - It can make a system simple. Especially, for the local testing purpose of application developers, I think fewer components and resources are a better choice.
- We can reduce unnecessary confusion and questions from actual users.
  - If there are some components that users cannot understand easily, they might confused and ask the support team `What are these components?` or `Do we need CFSSL in the production environment?` etc... To avoid such things, I think it would be better to keep the system (test environment) simple and minimal by default.

What do you think?
This comment does not prevent merging this PR, but I want to discuss it with you later!